### ISSUE TITLE
size check

### ISSUE THREAD

**Issue by ddkwork** [HyperDbg/gui#27]
todo

fix all size check fn

**Comment by ddkwork** [comment]
hello you， i can not sizeof go structure，can you help me？

![Screenshot_20221119_220418_com.realvnc.viewer.android.jpg](https://user-images.githubusercontent.com/19886504/202854561-a623df07-08df-4015-8e18-2ee218df3426.jpg)

**Comment by SinaKarvandi** [comment]
Hi,
Yes of course.

This is what I understand from your message. Probably there is nothing like "sizeof" in GO? and we need to export the size as a function? Am I right? 

Can you give an example of how to solve it? I will fix it then.

Thanks.

**Comment by ddkwork** [comment]
> Hi,
> Yes of course.
> 
> This is what I understand from your message. Probably there is nothing like "sizeof" in GO? and we need to export the size as a function? Am I right? 
> 
> Can you give an example of how to solve it? I will fix it then.
> 
> Thanks.

I think this case should not change c++ code,  maybe i should make a reflect func calc it, let me try again.

**Comment by ddkwork** [comment]
> Hi,
> Yes of course.
> 
> This is what I understand from your message. Probably there is nothing like "sizeof" in GO? and we need to export the size as a function? Am I right? 
> 
> Can you give an example of how to solve it? I will fix it then.
> 
> Thanks.

![Screenshot_20221121_012140_com.realvnc.viewer.android.jpg](https://user-images.githubusercontent.com/19886504/202916339-19c33272-e859-436d-8ba8-2cd860e55693.jpg)

size calced, but must init a value, do you have another way for use smaller code calc it ?

**Comment by ddkwork** [comment]
Working
